scripts/run_full_pipeline.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import json
+import subprocess
+
+
+def parse_args():
+    description = """This script can be used to run the full pipeline.
+    This includes surveyor jobs queuing downloader jobs, which in turn should queue processor jobs.
+    This includes Microarray experiments, RNA-Seq experiments, and transcriptome indices.
+    This script must be run from the top level directory of the refinebio repository.!"""
+    parser = argparse.ArgumentParser(description=description)
+
+    accession_help_text = """Specify the accession_code of the experiment to survey.
+    If creating a Transcriptome Index an organism name like 'Homo
+    Sapiens' should be supplied if the organism is in the main
+    division of Ensembl. If the organism is in any other division it
+    should be specified like 'Arabidopsis thaliana, EnsemblPlants'."""
+    parser.add_argument("accession_code", help=accession_help_text)
+
+    return parser.parse_args()
+
+
+def get_job_to_run():
+    completed_command = subprocess.run(
+        ["./foreman/run_surveyor.sh", "get_job_to_be_run"],
+        stdout=subprocess.PIPE,  # capture_output=True
+    )
+
+    # The JSON output is on the last line of the output, but it has a
+    # newline as well.
+    last_line = completed_command.stdout.decode("utf-8").split("\n")[-2].strip()
+    if completed_command.returncode == 0:
+        return json.loads(last_line)
+
+
+def run_job(job):
+    job_name = job["job_name"]
+    job_id = job["job_id"]
+    print(f"Running {job_name} Job with id {job_id}!")
+
+    image_name = ""
+    if job["job_type"] == "DownloaderJob":
+        subcommand = "run_downloader_job"
+        image_name = "downloaders"
+    else:
+        subcommand = "run_processor_job"
+
+        if job_name == "AFFY_TO_PCL":
+            image_name = "affymetrix"
+        elif job_name == "SALMON":
+            image_name = "salmon"
+        elif job_name == "ILLUMINA_TO_PCL":
+            image_name == "illumina"
+        elif job_name in ["TRANSCRIPTOME_INDEX_LONG", "TRANSCRIPTOME_INDEX_SHORT"]:
+            image_name = "transcriptome"
+        elif job_name == "NO_OP":
+            image_name = "no_op"
+
+    completed_command = subprocess.check_call(
```

Comment:
From [here](https://docs.python.org/3/library/subprocess.html#subprocess.check_call), it looks like `check_call` throws if the return code is nonzero, rather than returning the nonzero error code.

Also, does the output of this get sent to the terminal? That would be preferable, or at least somewhere that the output can be inspected.

Reply:
Oh good catch. I guess I don't gotta check the return codes.

The output does get sent to the terminal. It's a little bit chatty at the moment because the `run_job.sh` and `run_surveyor.sh` scripts rebuild the docker images every time but it is nice having them take care of that for you as you iterate.

I think that maybe in the future we could have the pipeline script look at what docker images would need to be built and build them once before kicking off work, but that's a larger refactor to the way the scripts work. In this PR I just wanted to get the flow working with the existing scripts and it already ended up being nontrivial.